build.gradle

```diff
@@ -17,8 +17,8 @@ def versionPatch = 0
 def versionBuild = 0
 
 android {
-  compileSdkVersion 22
-  buildToolsVersion '22.0.1'
+  compileSdkVersion 23
```

Comment:
I didn't change `targetSdkVersion` because I didn't want to deal with permission non-sense.